src/site/stages/build/page-builder/schemas/#page.js#

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
```

Comment:
We're definitely gonna have to convert all of this to camelcase at some point. It might make sense to do right out of the gate, so that the schemas align with the data structures in the templates. What do you think?

Reply:
That'll happen in the transformation step after this. The goal of this validation is to ensure that the data we get in the raw is what we expect.